reinforcement_learning/README.md

```diff
@@ -0,0 +1,93 @@
+# Reinforcement Learning
+
+This package contains all the resources to train a dialogue policy using reinforcement learning. It comprises:
+
+  * Gymnasium environments
+  * Training scripts
+
+## Gymnasium environments
+
+To use the environments, you first need to register them with Gymnasium, to do so you need to execute their attached script.
+For example, to register the `DialogueEnvMovieBot-v0` environment, you need to execute the following command:
+
+```bash
+python -m reinforcement_learning.environment.dialogue_env_moviebot
```

Comment:
Seems the `gym.register` calls were removed from the mentioned file(s).

Reply:
Good point. I removed this part and updated it with a short description of the available environment.